test/integration/phase_label_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
+
+class PhaseLabelTest < ActionDispatch::IntegrationTest
+  test "Beta phase label is displayed for a Service Manual Guide in phase 'beta'" do
+    ENV["FLAG_ENABLE_SERVICE_MANUAL"] = 'yes'
```

Comment:
This would be more graceful and less fragile using [ClimateControl](https://github.com/thoughtbot/climate_control) - what do you think?